01-js-intro-and-jquery/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>jquery exercise</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />       
+        <script src="jquery-2.1.4.min.js"></script>
+        <script src="app.js"></script>
+    </head> 
+    <body>
+        <header>
+
+        </header>
+        <section>
+            <h3>First section</h3>
+        </section>
+        <section>
+            <h3>Second section</h3>
+        </section>
+        <section>
+            <h3>Third section</h3>
+        </section>
+        <section class="hidden">
+            <h3 id="welcome">Hello world</h3>
+        </section>
+        <form>
+            <input name="name" id="name-txtbox" value="" class="alias" />
+        </form>
+        <button id="btn">Button</button>        
```

Comment:
I know it's not relevant for this topic, but be careful with IDs! :P There's no need to use them in any part of the page here.